src/backend/bs_srcserver

```diff
@@ -5843,6 +5862,7 @@ my $dispatches = [
   'POST:/source/$project cmd=undelete user:? comment:?' => \&undeleteproject,
   'POST:/source/$project cmd=copy user:? comment:? oproject:project withbinaries:bool? withhistory:bool? makeolder:bool? makeoriginolder:bool? resign:bool? noservice:bool?' => \&copyproject,
   'POST:/source/$project cmd=move oproject:project' => \&moveproject,
+  'POST:/source/$project cmd=freeze user:? comment:? requestid:num?' => \&freezeproject,
```

Comment:
@adrianschroeter Might be not so obvious where it is used:

BSRevision::addrev_meta_replace -> BSRevision::addrev_replace_common

Here it gets used:

https://github.com/openSUSE/open-build-service/blob/master/src/backend/BSRevision.pm#L280